daemon/algod/api/server/v2/handlers_test.go

```diff
@@ -39,3 +44,166 @@ func TestApplicationBoxesMaxKeys(t *testing.T) {
 	// Response size _not_ limited
 	require.Equal(t, uint64(math.MaxUint64), applicationBoxesMaxKeys(0, 0))
 }
+
+type tagNode struct {
```

Comment:
The `v2/test` package defines a testing environment for the handlers (mock ledger, db setup, etc). So if a test needs to use anything from that environment, it should likely be there.

Since there's still a `v2/handlers_test.go` file (which I think is confusing), my interpretation is that simpler unit tests that don't need the testing environment should be put there, and that's what I've done in this PR.